app/components/icon-catalog-form.ts

```diff
@@ -27,9 +27,9 @@ export default class IconCatalogFormComponent extends ImageUploadHandlerComponen
   }
 
   @action
-  async setImage(model: Icon, image: File) {
+  setImage(model: Icon, image: File) {
     super.setImage(model, image);
-    await this.args.icon.validate();
+    void this.args.icon.validate();
```

Comment:
what's the reason for this? it's a bit strange, could definitely use a commend

Reply:
Basically the linter was complaining, and there's no reason to await that promise like all the other validate promises, the validation just add something to the model itself so you are not waiting on it to return a value, you just want the promise to launch